src/components/AppConfig/CheckTypeItem.tsx

```diff
@@ -33,6 +33,7 @@ export const CheckTypeItem: React.FC<CheckTypeItemProps> = ({
                 <div className={s.switchWrapper}>
                   <Switch
                     value={!ignoreSteps.includes(step.stepID)}
+                    data-testid={`ignore-${step.stepID}`}
```

Comment:
nit : not sure how we do this nowadays but shouldn't this be a using a test ids object or enum so we don't have magic strings in both tests and app?